DiscordChatExporter.Cli/Commands/ExportGuildCommand.cs

```diff
@@ -60,8 +64,18 @@ await foreach (var channel in Discord.GetGuildChannelsAsync(GuildId, cancellatio
                 )
             )
             {
+                await console.Output.WriteAsync(
+                    $"  Found {progress++} threads.  {thread.Parent?.Name} - {thread.Name}"
+                        .PadRight(80)
+                        .Substring(0, 80) + '\r'
```

Comment:
What's the `PadRight`/`Substring` for? Not super clear.

Reply:
Also, we print one progress message for all channels, but multiple messages for threads?